src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -122,11 +122,14 @@ const SignOrExecuteForm = ({
   }
 
   // Sign transaction
-  const onSign = async (): Promise<string> => {
+  const onSign = async (): Promise<string | undefined> => {
     const [connectedWallet, createdTx, provider] = assertDependencies()
 
     // Smart contract wallets must sign via an on-chain tx
     if (await isSmartContractWallet(connectedWallet)) {
+      // If the first signature is a smart contract wallet, we have to propose w/o signatures
+      // Otherwise the backend won't pick up the tx
+      // The signature will be added once the on-chain signature is indexed
       const id = txId || (await proposeTx(createdTx))
```

Comment:
@Uxio0 FYI, as a temporary solution, we'll keep proposing txs w/o sigs when a smart contract wallet initiates a new transaction.
As per @rmeissner's comment:
<img width="398" alt="Screenshot 2022-12-08 at 12 37 56" src="https://user-images.githubusercontent.com/381895/206437551-52d3b403-84ca-47cd-ac6a-95ae26550561.png">

If you have a better idea, please let us know!

Reply:
No worries, as discussed on the weekly, we can go for now with allowing not signature transactions